src/containers/HomeBrowseCategory.jsx

```diff
@@ -1,26 +1,26 @@
-import React, { useEffect } from 'react';
+import React, { useEffect,useState } from 'react';
 import Button from '../components/Button';
 import Loading from '../components/Loading';
 import HomeCategoryProductCard from '../components/HomeCategoryProductCard';
-import { useGetAllCategoriesQuery } from '../states/api/apiSlice';
+import { useGetAllCategoriesQuery,useLazyGetAllProductsQuery } from '../states/api/apiSlice';
 import { setCategories } from '../states/features/categories/categorySlice';
 
 // BROWSE BY CATEGORY
 const BrowseByCategory = () => {
-  const props = {
-    image:
-      'https://res.cloudinary.com/nishimweprince/image/upload/v1685229667/ecommerce/c08012248_thyevv.png',
-    description: 'Brand new HP Envy 15, 256GB',
-    price: '$1499',
-    name: 'HP All-in-One',
-    category: 'Electronics',
-    rating: {
-      rating: 3,
-      count: 5
-    },
-    className:
-      'home_category_product_card flex flex-col justify-between my-4 w-fit max-w-1/5'
-  };
+  const[getAllProducts]=useLazyGetAllProductsQuery();
+  const [populars,setPopulars]=useState([]);
+  const category='BEAUTY';
```

Comment:
I suggest that you use the categories obtained from our API, and use the getCatName function defined in [`src/utils`](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-view-recommended-products-185218941/src/utils/categories.js#L1C1-L6C3) to get the category name dynamically using its (category) id.

The function looks like this:
```jsx
const getCatName = (id, arr) => {
  const category = arr.find((item) => {
    return item.id === id;
  });
  return category ? category.name : 'Category';
};
```